XamarinCommunityToolkit/Effects/EffectIds.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Xamarin.Forms;
+
+[assembly: ResolutionGroupName("Xamarin.CommunityToolkit.Effects")]
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	sealed class EffectIds
+	{
+        static string effectResolutionGroupName = $"{nameof(Xamarin)}.{nameof(CommunityToolkit)}.{nameof(Effects)}";
```

Comment:
```suggestion
        static string EffectResolutionGroupName = $"{nameof(Xamarin)}.{nameof(CommunityToolkit)}.{nameof(Effects)}";
```

Reply:
@jfversluis will we use this approach to do docs? Or will we set up the mdocs with a separate xml file?

If the answer from @jfversluis is a `yes`, can you add a `summary` for this?